frame/support/procedural/src/construct_runtime/mod.rs

```diff
@@ -425,3 +442,38 @@ fn decl_integrity_test(scrate: &TokenStream2) -> TokenStream2 {
 		}
 	)
 }
+
+/// Assign index to each modules using same rules as rust for fieldless enum.
+/// I.e. implicit are assigned number incrementedly from last explicit or 0.
+fn assign_implicit_index(modules: &mut Vec<ModuleDeclaration>) -> syn::Result<()> {
```

Comment:
Would like to have 1 or 2 ui tests for this.

Reply:
added 👍